compliance_checker/suite.py

```diff
@@ -71,10 +74,11 @@ def _get_generator_plugins(cls):
         Return a list of classes from external plugins that are used to
         generate checker classes
         """
-
+        # NOTE: updated to not use pkg_resources, but
+        #       not tested -- it is ever used?
         if not hasattr(cls, "suite_generators"):
-            gens = working_set.iter_entry_points("compliance_checker.generators")
-            cls.suite_generators = [x.resolve() for x in gens]
+            gens = impmd.entry_points(group='compliance_checker.generators')
```

Comment:
```suggestion
            gens = importlib_metadata.entry_points(group='compliance_checker.generators')
```

Reply:
See above, by giving it a name, we don't have to go in and change code when we no longer need the backport -- taste once again.

Reply:
Compliance-checker code us huge and quite complex to read. I'd prefer to have the full import name here so we can read it and know what it is right away. I understand where you are going and I do that in my own projects. But this one is maintained by many people and I'd rather not rock the boat with style changes for minor things.